less/footer.less

```diff
@@ -8,5 +8,11 @@
     color: @font-color;
     font-size: 1.6rem;
     padding: 20px;
+    width: 100%;
+  
+
+  @media@phone {
+    font-size: 3vw;
```

Comment:
Be VERY careful of using the 'vw' property. It really should only be used in very specific use cases. Using rem here would have been fine.